virt/release_notes/virt-4-14-release-notes.adoc

```diff
@@ -206,32 +197,58 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 //CNV-20106 BZ 2092412
 * Virtual machines created from common templates on a Single Node OpenShift (SNO) cluster no longer display a `VMCannotBeEvicted` alert when the template’s default eviction strategy is LiveMigrate. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2092412[*BZ#2092412*])
 
+//CNV-18526 BZ 2089301
+* Windows 11 virtual machines now boot on clusters running in link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/security_hardening/index#con_federal-information-processing-standard-fips_assembly_installing-the-system-in-fips-mode[FIPS mode]. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2089301[*BZ#2089301*])
+
+//BZ 2151169
+* In a heterogeneous cluster with different compute nodes, virtual machines that have HyperV Reenlightenment enabled can be scheduled on nodes that do not support timestamp-counter scaling (TSC) or have the appropriate TSC frequency. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2151169[*BZ#2151169*])
+
+* When you use two pods with different SELinux contexts, VMs with the `ocs-storagecluster-cephfs` storage class no longer fail to migrate. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2092271[*BZ#2092271*])
+
+//OCPBUGS-8398
+* If you stop a node on a cluster and then use the Node Health Check Operator to bring the node back up, connectivity to Multus is retained. (link:https://issues.redhat.com/browse/OCPBUGS-8398[*OCPBUGS-8398*])
+
+* When restoring a VM snapshot for storage whose binding mode is `WaitForFirstConsumer`, the restored PVCs no longer remain in the `Pending` state and the restore operation proceeds. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2149654[*BZ#2149654*])
+
 [id="virt-4-14-known-issues"]
 == Known issues
 
-//CNV-24889
+//CNV-24889 (Avital mentions possible RN in CNV-32472)
 
-//CNV-20106 Release note: Remove known issue 2092412/PREVIEW (depends on CNV-20086)
+//Known issues from 4.13 (and several carried over from 4.12) begin here.Leave in all for now; will be re-reviewed for 4.14.
 
+//4.13 Leave in per Simone
+* Uninstalling {VirtProductName} does not remove the `feature.node.kubevirt.io` node labels created by {VirtProductName}. You must remove the labels manually. (link:https://issues.redhat.com/browse/CNV-22036[*CNV-22036*])
```

Comment:
This RN is duplicated on line 272, so you can remove one of them

Reply:
Done